src/package-linker.js

```diff
@@ -147,6 +157,7 @@ export default class PackageLinker {
         },
       });
     }
+    this.reporter.info('skipped ' + skipped.length + ' modules because they are already installed');
```

Comment:
This shouldn't be a string here, it should use `this.reporter.lang` like everywhere else.